bot/cogs/fun.py

```diff
@@ -199,6 +199,43 @@ async def lizard(self, ctx) -> None:
                 else:
                     await ctx.send(f"Something went Boom! [status : {lizr.status}]")
 
+    @commands.command(aliases=["leet"])
+    async def Leetify(self, ctx):
+        Leetters = {'a':['A','4'], 'b':['b', 'B'], 'c':['C', 'c'], 'd':['D', 'd'],
+                        'e':['E', '3'], 'f':['F', 'f'], 'g':['G', 'g'], 'h':['H', 'h'],
+                        'i':['1', 'I', 'i'], 'j':['j', 'J'], 'k':['K', 'k'], 'l':['L', 'l'],
+                        'm':['M', 'm'], 'n':['n', 'N'], 'o':['O', '0'], 'p':['P', 'p'],
+                        'q':['Q', 'q'], 'r':['R', 'r'], 's':['5', 'S'], 't':['t', 'T'],
+                        'u':['U', 'u'], 'v':['v','V'], 'w':['W', 'w'], 'x':['X', 'x'],
+                        'y':['y', 'Y'], 'z':['Z', 'z']}
+        content = [i.lower()+' ' for i in [i for i in ctx.message.content.split(" ") if i.lower() != ">leet" and i.lower() != ">leetify"]]
+        strContent = ''
+        for i in content:
+            strContent += i
+        RefContent = ''
+        prevMD = ''
+        for i in strContent:
+            if i in Leetters.keys():
+                MDlist = ['*', '**', '***', '__', '', '`']
+                random.seed(random.randint(421, 294244))
+                if prevMD == '':
+                    MDlist.remove('')
+                elif '*' in prevMD:
+                    MDlist.remove("*")
+                    MDlist.remove("**")
+                    MDlist.remove("***")
+                elif prevMD == '__':
+                    MDlist.remove("__")
+                elif prevMD == '`':
+                    MDlist.remove("`")
+                ChosenMD = random.choice(MDlist)
+                prevMD = ChosenMD
+                RefContent += ChosenMD+random.choice(Leetters[i])+ChosenMD
```

Comment:
We follow the usual `flake8` python syntax which requires whitespaces around arithmetic operators, so this line should look like this:
```suggestion
                RefContent += ChosenMD + random.choice(Leetters[i]) + ChosenMD
```

Reply:
This suggestion also applies to all occurrences of breaking this standard flake8 syntax concerning whitespaces around these operator, for example you also violate it on line 213 with the complicated list comprehention:

```py
 content = [i.lower()+' ' for i in [i for i in ctx.message.content.split(" ") if i.lower() != ">leet" and i.lower() != ">leetify"]]
```

These conventions need to be followed everywhere, the suggestion was only 1 example of it